### PR TITLE
Update FAQ.md

### PR DESCRIPTION
**Pull request by filimonov**
Add list of shortcuts to FAQ (hitting F1 requires shifting the focus).

### CODE REVIEW

File: Documentation/FAQ.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ highly not recommended), you can use `chdig --connection prod`.
   [1]: https://github.com/ClickHouse/ClickHouse/pull/45715
   [2]: https://github.com/ClickHouse/ClickHouse/pull/46480
 
+### What are the shortcuts supported?
+
+![image](https://github.com/azat/chdig/assets/1549571/ae5719e1-3528-4676-938b-b1bed84b6ed7)
```
Comment (azat) [inline Documentation/FAQ.md:43]: Let's write it in markdown instead of screenshot

**Comment by filimonov** [comment]
In markdown that looks worse, but ok :) (at least searchable)

**Comment by azat** [comment]
>at least searchable

Yep!

Can you also resolve the conflicts

File: Documentation/FAQ.md [hunk 2]
```diff
@@ -38,6 +38,55 @@ highly not recommended), you can use `chdig --connection prod`.
   [1]: https://github.com/ClickHouse/ClickHouse/pull/45715
   [2]: https://github.com/ClickHouse/ClickHouse/pull/46480
 
+### What are the shortcuts supported?
+
+for chdig v24.3.1
```
Comment (azat) [inline Documentation/FAQ.md:43]:
```suggestion
```

Let's remove this line and hope that we will not forget to keep it in sync.

File: Documentation/FAQ.md [hunk 3]
```diff
@@ -38,6 +38,55 @@ highly not recommended), you can use `chdig --connection prod`.
...
+
```
Comment (azat) [inline Documentation/FAQ.md:44]:
```suggestion
```

And this one

**Review by azat — APPROVED** [review]

**Pull request merged**